src/Calendar.js

```diff
@@ -568,7 +568,19 @@ class Calendar extends React.Component {
      day: PropTypes.node,
      agenda: PropTypes.node,
      showMore: PropTypes.func
-   })
+   }),
+
+   /**
+    * Ref for the calendar's root div. Can be either the old ref style (function/string)
+    * or the new ref style (React.createRef).
+    */
+   ref: PropTypes.oneOfType([
+     PropTypes.func,
+     PropTypes.string,
```

Comment:
This is considered legacy and shouldn't be supported. https://zhenyong.github.io/react/docs/more-about-refs.html#the-ref-string-attribute

Reply:
I know, but it's still a valid type of ref. I was just covering the possible types of refs, I'm not going to use any of them in `hippo` code other than React.createRef.

Reply:
I don't think it's this component's job to disallow older ref types via proptypes, it's our job as hippo consumers to be aware that createRef is the current way to create refs.